brouter-routing-app/build.gradle

```diff
@@ -94,3 +99,17 @@ dependencies {
     implementation project(':brouter-util')
 
 }
+
+task generateProfiles(type: Exec) {
+    commandLine = "../misc/scripts/generate_profile_variants.sh"
```

Comment:
This executes the `generate_profile_variants.sh` to generate all profiles. I think that's not the best way because the build now requires bash & sed.